### ISSUE TITLE
Docker run error

### ISSUE THREAD

**Issue by Jarvis-K** [deepkit/deepkit#15]
I tried to use the deepkit with clusters. After configuring the `deepkit.yml` as follows:
```
image: xxx
command: 
    - echo hell
```
I run it on the server, but got the error:
```
Success fully built 47df7b7037f5
Successfully tagged deepkit_ test_ 3980a55c32a6d7a01ac7 7537ff8495dc: latest
start docker deepkit test 3980a5 5c32a6d7a01ac77537ff8495dc : echo hell
run
failed Error: ( HTTP code 400 ) unexpected
invalid environment variable :
( HTTP code 400 )
unexpected
invalid environment variable:
```
Can u help me to figure out how to fix it?

**Comment by marcj** [comment]
Which operating system do you use and which Docker version? Did you set manually Docker environment variables in the cluster node settings? It might be that at 
https://github.com/deepkit/deepkit/blob/master/packages/cli/src/task.ts#L607-L616
there are some invalid values given.

**Comment by Jarvis-K** [comment]
Thanks for your replay.
I am currently using Docker 19.03.6 in Ubuntu 16.04; I did not set any env variables. 
How can I find out which environment value is invalid? I tried add `mkdir job`, the issue still exist. Or have you tested some images that work well with deepkit? With it I can verify whether the problem is caused by the docker image or other settings of my cluster.

**Comment by marcj** [comment]
All images should work perfectly fine. Which did you use? So you cluster node server has Ubuntu 16.04, right?

You can try starting your image by using `docker run imagename` on your server directly in order to see if that works correctly.

**Comment by Jarvis-K** [comment]
Yes. I tried my own image and the ubuntu official images. They work fine when start on the server. Is it possible to inspect every command the deepkit runs? Or the detailed log about which command failed?

**Comment by marcj** [comment]
> Is it possible to inspect every command the deepkit runs?

Not yet. I'm currently working on a debug mode so we get valuable insights in such cases.

> Or the detailed log about which command failed? 

I'm pretty sure one of those variables I linked are wrong, but I don't know yet in which case that happens and which exactly are malformed. I'm afraid you won't easily find out which either since you can't not easily change the source code to inject debug statements.
